src/sgp/SGP.cc

```diff
@@ -506,15 +436,15 @@ int main(int argc, char* argv[])
 		GCM = NULL;
 
 		return EXIT_SUCCESS;
-	} catch (...) {
-		try {
-			TerminationHandler();
-		} catch (...) {
-			// If you ever see return code 27, try to set a breakpoint here
-			return 27;
-		}
-		return EXIT_FAILURE;
-	}
+	// } catch (...) {
```

Comment:
Is this a remnant from the development phase or do you really want to check this in?

Reply:
Yep, this should not be checked in. We really should have a way to define this via CMake. Otherwise most errors will end up here and not in the debugger, or maybe I am missing something?